src/lib/configs/project/dockerCompose.ts

```diff
@@ -269,11 +329,11 @@ async function genDockerCompose(
         }),
       ),
     },
-    secrets: Object.fromEntries(
-      peers.map(({ name, path: file }) => {
-        return [name, { file }];
+    secrets: Object.fromEntries([
+      ...peers.map(({ name, relativeSecretFilePath: file }) => {
+        return [name, { file }] as const;
       }),
-    ),
+    ]),
```

Comment:
Is there are some ts errors that made you to use spread operator here?
Probably should work without spread.